service/src/main/java/bio/terra/axonserver/service/cromwellworkflow/CromwellWorkflowService.java

```diff
@@ -280,6 +290,34 @@ public CromwellApiLabelsResponse getLabels(UUID workflowId)
         .labels(CROMWELL_CLIENT_API_VERSION, workflowId.toString());
   }
 
+  public Map<String, String> parseInputs(UUID workspaceId, String gcsPath, BearerToken token)
+      throws IOException, InvalidWdlException {
+    // 1) Get the WDL file and write it to disk
+    try (AutoDeletingTempFile tempWdlFile = new AutoDeletingTempFile("workflow-main-wdl", "wdl")) {
+      InputStream resourceObjectStream = fileService.getFile(token, workspaceId, gcsPath, null);
+      DefaultPath cromwellPath = DefaultPathBuilder.build(tempWdlFile.getFile().toPath());
+      Files.copy(
+          resourceObjectStream,
+          tempWdlFile.getFile().toPath(),
+          StandardCopyOption.REPLACE_EXISTING);
+
+      // 2) Call Womtool's input parsing method
+      boolean showOptionals = true;
```

Comment:
tiny nit: as I've recently learned intellij shows the parameter name when you call a function with a static value, so from what I can tell the java approach to this is just: Inputs.inputsJson(cromwellPath, true)